app/issue.py

```
@@ -0,0 +1,23 @@
+        return
+
+    url = f"https://api.github.com/repos/{repo}/issues"
+    headers = {
+        "Authorization": f"Bearer {token}",
+        "Accept": "application/vnd.github+json",
+    }
+    payload = {"title": title, "body": body}
+
+    r = requests.post(url, headers=headers, json=payload, timeout=10)
+    if r.status_code >= 300:
+        logger.warning(f"Failed to create issue: {r.status_code} {r.text[:200]}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)